cowprotocol/services
# Trade verification in e2e tests

# Changes
This PR enables trade verification in all e2e tests.
Note that we still only verify quotes but not yet drop quotes that failed verification.

## How to test
e2e tests still pass
Ran tests locally with additional debug logs to see that we indeed see some quotes getting verified.
In fact there is only 1 test left which produces a quote that can not be verified. This is a test for partially fillable orders where we create an order for more sell_token than we have so it's expected that the verification fails.

I'm unhappy with how ugly tests look that are not covered by the regular `start_protocol()` function.
Unless people object I would merge the tests as they are but we should probably find a nicer way to configure these tests in the long term.
